packages/venia-concept/src/components/Pagination/pagination.js

```diff
@@ -27,11 +27,6 @@ class Pagination extends Component {
     };
 
     componentDidMount() {
-        // updateTotalPages pushes the current page count of a category query to
-        // redux so it knows how many tiles to render even in the Query
-        // component's loading state
-        const { updateTotalPages, totalPages } = this.props.pageControl;
-        updateTotalPages(totalPages);
```

Comment:
We don't need to do this anymore?

Reply:
Nope. We're grabbing this from the queryResult now, instead of calculating ourselves in `category.js` and prop drilling, so it's already set.